.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        ghc: ["9.2", "9.0", "8.10"]
+        ghc: ["9.6", "9.4", "9.2"]
```

Comment:
@CristhianMotoche if we are going to exclude `9.0` from the build matrix, I think we should bump the version constraint for `base` accordingly

Reply:
That's a good observation @sestrella I will preserve the testing against GHC 8.10 since dotenv can be used in  projects with that version of GHC without any inconveniences. Although, I do need to change the base boundaries since [GHC 8.10 is shipped with base 4.14](https://wiki.haskell.org/Base_package).